Tiltfile

```diff
@@ -7,8 +7,8 @@ repo = local_git_repo('.')
 live_update = [
   sync('internal', '/go/src/github.com/windmilleng/tilt/internal'),
   sync('web', '/go/src/github.com/windmilleng/tilt/web'),
-  run('make build-js'),  # trigger=['web'] (when support directory/glob triggers)
-  run('make install-sail'),  # trigger=['internal'] (when support directory/glob triggers)
+  run('make build-js', trigger=['web/']),
```

Comment:
What happens if you leave the slash off here?

Reply:
it also works -- i just wanted it to be visually clear that it's a directory. i don't feel strongly, do you have thoughts about what's prettier here?

Reply:
Nope just wanted to make sure it still worked without the slash. :)